src/common/src/array/arrow/arrow_impl.rs

```diff
@@ -505,6 +505,11 @@ pub trait FromArrow {
             Int16 => DataType::Int16,
             Int32 => DataType::Int32,
             Int64 => DataType::Int64,
+            UInt8 => DataType::Int16,
```

Comment:
Do we need `Int8` here? If yes, why is it not caught by tests?

Reply:
Uncover by test.